src/test/scala/uk/gov/nationalarchives/omega/api/repository/TestRepository.scala

```diff
@@ -161,4 +162,213 @@ class TestRepository extends AbstractRepository {
           )
         )
     }
+
+  override def getRecordConceptEntity(recordConceptId: String): Try[List[RecordConceptEntity]] =
+    Try(
+      List(
+        RecordConceptEntity(
+          new URI(s"${BaseURL.cat}/COAL.2022.N373.P"),
+          new URI("http://www.nationalarchives.gov.uk/ont.physical-record"),
```

Comment:
Should the base URI here come from a constant?

Reply:
Moved to a constant.